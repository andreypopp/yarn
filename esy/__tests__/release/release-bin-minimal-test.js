/**
 * @flow
 */

jest.setTimeout(200000);

import testMinimal from './fixtures/minimal/test';

const testCase = testMinimal({releaseType: 'bin'});

test(testCase.description, testCase.test);
afterAll(testCase.cleanUp);
