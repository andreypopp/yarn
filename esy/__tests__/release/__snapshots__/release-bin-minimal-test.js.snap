// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`installation 1`] = `
"<root>
| bin
| | minimal
| etc
| lib
| | node_modules
| | | minimal
| | | | .bin
| | | | | minimal
| | | | package.json
| | | | postinstall.sh
| | | | prerelease.sh
| | | | records
| | | | | recordedClientBuildStorePath.txt
| | | | | recordedClientInstallStorePath.txt
| | | | | recordedCoppiedArtifacts.txt
| | | | | recordedServerBuildStorePath.txt
| | | | rel
| | | | | 3.x.x______________________________
| | | | | | i
| | | | | | | minimal-0.1.0
| | | | | | | | bin
| | | | | | | | doc
| | | | | | | | etc
| | | | | | | | lib
| | | | | | | | man
| | | | | | | | sbin
| | | | | | | | share
| | | | | node_modules
| | | | | | .cache
| | | | | | | _esy
| | | | | | | | build-eject
| | | | | | | | | bin
| | | | | | | | | | fastreplacestring.cpp
| | | | | | | | | | fastreplacestring.exe
| | | | | | | | | | realpath
| | | | | | | | | | realpath.c
| | | | | | | | | | render-env
| | | | | | | | | | runtime.sh
| | | | | | | | | command-env
| | | | | | | | | eject-env
| | | | | | | | | final-install-path-set.txt
| | | | | | | | | final-install-path-set.txt.in
| | | | | | | | | Makefile
| | | | | | | | | sandbox.sb.in
| | | | | | | | store
| | | | | | | | | b
| | | | | | | | | i
| | | | | | | | | s
| | | | | package.json
| | | | | test.js
| | | | | yarn.lock"
`;

exports[`installation stdout 1`] = `
"/tmp/TMPDIR/npm/bin/minimal -> /tmp/TMPDIR/npm/lib/node_modules/minimal/.bin/minimal

> minimal@0.1.0 postinstall /tmp/TMPDIR/npm/lib/node_modules/minimal
> ./postinstall.sh

*** Decompressing artefacts...
*** Relocating artefacts to the final destination...
+ minimal@0.1.0
added 1 package in X.XXXs"
`;

exports[`release 1`] = `
"<root>
| bin-darwin
| | .bin
| | | minimal
| | package.json
| | postinstall.sh
| | prerelease.sh
| | records
| | | recordedClientBuildStorePath.txt
| | | recordedCoppiedArtifacts.txt
| | | recordedServerBuildStorePath.txt
| | rel.tar.gz
| | relBinaries
| | | i
| | | | minimal-0.1.0.tar.gz"
`;

exports[`release stdout 1`] = `
"*** Creating bin-type release for minimal...
*** Installing esy for the release...
*** Installing dependencies...
*** Building the release..."
`;
